Src/LiquidProjections.NEventStore/NEventStoreAdapter.cs

```diff
@@ -299,7 +335,7 @@ private void StartPreloadingNextPage(long previousCheckpoint, string subscriptio
                         $"Failed loading transactions after checkpoint {previousCheckpoint} from NEventStore",
                         exception);
 
-                return new Page(previousCheckpoint, new List<Transaction>());
+                throw;
```

Comment:
By returning an empty page we also reused the polling interval mechanism. Now when we just throw, we will retry immediately. Not sure if it's good or bad.